CMakeLists.txt

```diff
@@ -248,6 +254,46 @@ else()
   )
 endif()
 
+# clang-tidy ---------------------------------------------------------------------------------------
+
+option(CS_RUN_CLANG_TIDY "Run clang-tidy with the compiler." Off)
```

Comment:
For the sake of consistency, I would suggest to call this option `COSMOSCOUT_RUN_CLANG_TIDY`.